todo_app/app.py

```diff
@@ -9,51 +9,90 @@
 from flask_login import current_user
 import os
 import requests
-import json
 from oauthlib.oauth2 import WebApplicationClient
+import logging
+from loggly.handlers import HTTPSHandler 
+from logging import Formatter
+import logging.config
+import time
+
+# create logger
+logger = logging.getLogger('todo_app')
+logging.config.fileConfig('python.conf')
+logging.Formatter.converter = time.gmtime
+logger.setLevel(logging.DEBUG)
+ 
+# create console handler and set level to debug
+ch = logging.StreamHandler()
 
+ch.setLevel(logging.DEBUG)
+# create formatter
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+# add formatter to ch
+ch.setFormatter(formatter)
+# add ch to logger
+logger.addHandler(ch)
```

Comment:
I don't think you need these lines anymore - you're successfully configuring the logger in create_app, right?